ios/Plugin/NewRelicCapacitorPluginPlugin.swift

```diff
@@ -428,6 +428,11 @@ public class NewRelicCapacitorPluginPlugin: CAPPlugin {
             "fatal": isFatal,
             "stackTraceElements": stackFramesArr
         ]
+        if let customAttributes = call.getObject("attributes") {
+            for (key, value) in customAttributes {
+                attributes[key] = value
+            }
```

Comment:
This is a compilation error. You'll have to change `attributes` to be mutable.

`error: cannot assign through subscript: 'attributes' is a 'let' constant
                attributes[key] = value
                ~~~~~~~~~~^`

Reply:
Changed 'let' to 'var' for attributes variable